src/MPS.jl

```diff
@@ -540,20 +541,35 @@ function mixed_canonize!(tn::AbstractMPO, orthog_center)
     return tn
 end
 
+LinearAlgebra.normalize(ψ::AbstractMPO, site) = normalize!(copy(ψ), site)
+
 LinearAlgebra.normalize!(ψ::AbstractMPO; kwargs...) = normalize!(form(ψ), ψ; kwargs...)
+LinearAlgebra.normalize!(ψ::AbstractMPO, site) = normalize!(form(ψ), ψ; at=site)
 
 function LinearAlgebra.normalize!(::NonCanonical, ψ::AbstractMPO; at=Site(nsites(ψ) ÷ 2))
     tensor = tensors(ψ; at)
     tensor ./= norm(ψ)
     return ψ
 end
 
-LinearAlgebra.normalize!(ψ::AbstractMPO, site::Site) = normalize!(mixed_canonize!(ψ, site); at=site)
-
 function LinearAlgebra.normalize!(config::MixedCanonical, ψ::AbstractMPO; at=config.orthog_center)
     mixed_canonize!(ψ, at)
     normalize!(tensors(ψ; at), 2)
     return ψ
 end
 
-# TODO function LinearAlgebra.normalize!(::Canonical, ψ::AbstractMPO) end
+function LinearAlgebra.normalize!(config::Canonical, ψ::AbstractMPO; at=nothing)
+    if isnothing(at) # Normalize all λ tensors
+        normalizer = (norm(ψ))^(1 / (nsites(ψ) - 1))
+
+        for i in 1:(nsites(ψ) - 1)
+            λ = tensors(ψ; between=(Site(i), Site(i + 1)))
+            replace!(ψ, λ => λ ./ normalizer)
+        end
+    else
+        λ = tensors(ψ; between=at)
+        replace!(ψ, λ => λ ./ norm(ψ))
```

Comment:
shouldn't you just normalize the corresponding $\Lambda$ at that bond?
```suggestion
        λ = tensors(ψ; between=at)
        λ ./= norm(λ)
```